### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mhyryak**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -4,7 +4,31 @@ describe('getHumanAge', () => {
...
+
+  test('should return an array', () => {
+    expect(Array.isArray(getHumanAge(0, 0))).toBe(true);
+  });
+
+  test('should return an array of length 2', () => {
+    expect(getHumanAge(0, 0)).toHaveLength(2);
+  });
+
+  test('should return correct human ages for cats and dogs', () => {
```
Comment (djtyrf312) [inline src/getHumanAge.test.js:18]: split this test into different tests

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -4,7 +4,31 @@ describe('getHumanAge', () => {
...
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+    expect(getHumanAge(23, 23)).toEqual([1, 1]);
+    expect(getHumanAge(24, 24)).toEqual([2, 2]);
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
+    expect(getHumanAge(28, 28)).toEqual([3, 2]);
+    expect(getHumanAge(100, 100)).toEqual([21, 17]);
+  });
+
+  test('should handle edge cases correctly', () => {
```
Comment (djtyrf312) [inline src/getHumanAge.test.js:29]: split this test into different tests, remove extra comment and next time try to do it by yourself but not with chatGPT)

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
check comments in pullrequest

File: src/getHumanAge.test.js (hunk 2 above)
Comment (mhyryak) [reply src/getHumanAge.test.js:29]: I didn't use chat GPT that often, this topic is quite difficult for beginners. This time my friend who is a developer in the Sii company helped me.  For the previous tests which you refused, he helped me to write. A new one I did by myself with info from the morning lesson) 
Thank you for your advice!

**Review by KarolVlasova — APPROVED** [review]
